core/src/debug/assets/mobile-core.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
I think this file can be removed as we have renamed it and it's now duplicated. Is that right?